salt/states/boto_vpc.py

```diff
@@ -125,14 +125,81 @@ def present(name, cidr_block, instance_tenancy=None, dns_support=None,
     return ret
 
 
+def subnet_present(name, vpc_id, cidr_block, availability_zone=None, tags=None, region=None,
+                   key=None, keyid=None, profile=None):
+    '''
+    Ensure subnet exists.
+    .. versionadded:: 2014.7.3
```

Comment:
Since this change is going into develop (and also since 2014.7 is in feature freeze), this versionadded should be `Beryllium`.